extensions/amp-access-scroll/0.1/amp-access-scroll.css

```diff
@@ -15,34 +15,75 @@
  */
 
 .amp-access-scroll-bar {
-  height: 44px;
   position: fixed;
-  left: 0;
-  width: 100%;
   background: transparent;
   z-index: 2147483647;
+
   bottom: 0;
+  height: 59px;
+  left: auto;
+  right: 0px;
+  width: 47px;
 }
 
-.amp-access-scroll-audio {
-  position: fixed;
-  background-color: transparent;
+.amp-access-scroll-bar.amp-access-scroll-holdback {
+  height: 44px;
+  left: 0;
+  right: 0;
+  width: 100%;
+}
+
+.amp-access-scroll-sheet.amp-access-scroll-holdback {
   position: fixed;
+  background: transparent;
   z-index: 2147483647;
-  display: block;
 
   /* mobile */
   bottom: 44px;
   height: 100px;
   left: 0;
+  right: 0;
   width: 100%;
 }
+
 @media (min-width: 600px) {
-  .amp-access-scroll-audio {
+  .amp-access-scroll-sheet.amp-access-scroll-holdback {
     bottom: 63px;
     height: 56px;
     left: auto;
     right: 16px;
     width: 475px;
   }
 }
+
+.amp-access-scroll-sheet {
```

Comment:
could you put this classname above the modifications to it?

Reply:
Done.